pkg/cmd/server/kubernetes/node/options/options.go

```diff
@@ -89,13 +83,6 @@ func ComputeKubeletFlags(startingArgs map[string][]string, options configapi.Nod
 		setIfUnset(args, "iptables-masquerade-bit", "0")
 	}
 
-	if network.IsOpenShiftNetworkPlugin(options.NetworkConfig.NetworkPluginName) {
```

Comment:
This surprises me.  How are you determining the network plugin now?

Reply:
Ansible handles this.  The default is kubenet.